internal/services/bot/bot_channel_test.go

```diff
@@ -11,9 +11,10 @@ func TestAccBotChannelsRegistration(t *testing.T) {
 	// Azure only being able provision against one app id at a time
 	acceptance.RunTestsInSequence(t, map[string]map[string]func(t *testing.T){
 		"basic": {
-			"basic":    testAccBotChannelsRegistration_basic,
-			"update":   testAccBotChannelsRegistration_update,
-			"complete": testAccBotChannelsRegistration_complete,
+			"basic":                      testAccBotChannelsRegistration_basic,
+			"update":                     testAccBotChannelsRegistration_update,
+			"complete":                   testAccBotChannelsRegistration_complete,
+			"isStreamingEndpointEnabled": testAccBotChannelsRegistration_isStreamingEndpointEnabled,
```

Comment:
Can we also rename the test to just `streamingEndpointEnabled`?

Reply:
Updated.